src/main/java/com/whitesource/cure/Encoder.java

```diff
@@ -88,9 +88,9 @@ public static String crlfApacheEncoder(@NonNull final String content) {
    * @param content {@link Object} contains the content.
    * @return encoded JavaScript block.
    */
-  public static String forJavaScriptBlock(@NonNull final String content) {
+  public static String forJavaScriptBlock(@NonNull final Object content) {
```

Comment:
The name should reflect that it is an encoder for xss, 
Option 1: create an addional class XssEncoder
Option 2: add "Xss" at the end of each relevant method name